src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -249,6 +254,10 @@ export class HaIntegrationCard extends LitElement {
     this.classList.remove("highlight");
   }
 
+  private _toDocumentation() {
+    window.open(`https://www.home-assistant.io/integrations/${this.domain}`);
```

Comment:
This will not work for custom integrations, the documentation link is in the manifest, check <https://github.com/home-assistant/frontend/blob/dev/src/panels/config/info/integrations-card.ts#L63>

Also, you should use `<a href>` instead of `window.open`.

Reply:
I will make the change to an anchor, also I was not aware of the issue custom integrations would impose. I will use manifest for the link. 

Sorry. I am a new dev and love this project, so I figured I would tackle some of the issues. Thank you so much for your input. I will make the changes.